mappings/net/minecraft/datafixer/fix/NonStretchingPackedArray.mapping

```diff
@@ -0,0 +1,24 @@
+CLASS net/minecraft/class_5298 net/minecraft/datafixer/fix/NonStretchingPackedArray
```

Comment:
Is this a util class or an actual data fix?

If it's a util class it should go in util package.

Reply:
A utility array that is only used by data fix. Hmm no other such helper object exist in fix package, sure